nepalingo-web/src/pages/Home/Home.tsx

```diff
@@ -3,18 +3,28 @@ import { Link } from "react-router-dom";
 import logo from "../../assets/logo.png";
 import ReactGA from 'react-ga4';
 import { useAuth } from "../../components/userAuth/AuthContext";
+import GreetingCard from "../../components/GreetingCard";
 
 const Home: React.FC = () => {
-  ReactGA.send({ hitType: "pageview", page: window.location.pathname, title: "home"});
+  ReactGA.send({ hitType: "pageview", page: window.location.pathname, title: "home" });
   const { user } = useAuth();
 
+  const userName = 'user';
+
+
+
   return (
     <div className="flex flex-col items-center justify-between h-screen bg-gradient-to-r from-black via-gray-800 to-black text-white p-10">
       <div className="mt-10">
         <h1 className="text-5xl font-bold text-center">
           Hello {user?.user_metadata?.username}, welcome to Nepalingo!
```

Comment:
you can remove this logic here

Reply:
![CleanShot 2024-07-17 at 22 24 20@2x](https://github.com/user-attachments/assets/9f111e35-8a4b-4b4e-8e0f-94e92c0c96f4)
If we don't remove it, we'll have two greeting messages 😂 
I like the `welcome to Nepalingo!` part though, can we remove this line, and add it to your greeting card?